src/test/java/calculator/data/type/OperationTest.java

```diff
@@ -0,0 +1,74 @@
+package calculator.data.type;
+
+import calculator.view.Input;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class OperationTest {
+
+    @Test
+    @DisplayName("덧셈 테스트")
+    public void addTest(){
+        int num1 = 3;
+        int num2 = 5;
+        int expectedValue = 8;
+        Operation addition = Operation.ADDITION;
+        int result = addition.apply(num1, num2);
+        Assertions.assertThat(result).isEqualTo(expectedValue);
```

Comment:
간단한 테스트 코드이니 지역변수 없이 테스트 가능할 것 같네요
```suggestion
        Operation addition = Operation.ADDITION;
        int result = addition.apply(3, 5);
        Assertions.assertThat(result).isEqualTo(8);
```